src/components/DittoComponent.tsx

```diff
@@ -3,17 +3,26 @@ import { useDittoComponent } from "../hooks/useDittoComponent";
 import { DittoComponentLibraryProps } from "./Ditto";
 
 export const DittoComponent = (props: DittoComponentLibraryProps) => {
-  const { children, componentId, variables, count } = props;
+  const { children, componentId, variables, count, richText } = props;
 
   const text = useDittoComponent({
     componentId,
     variables: variables || {},
     count,
+    richText
   });
 
   if (typeof text === "string" && typeof children === "function") {
+    if (richText) {
+      return <>{children(text)}</>;
+    }
+
     return <>{children(text)}</>;
   }
 
+  if (richText) {
+    return <span dangerouslySetInnerHTML={{__html: text || "" }} />;
```

Comment:
I believe `dangerouslySetInnerHTML` is inevitable here, but if you have any alternative suggestions I'd be glad to hear it.

Reply:
hmm....i wonder if this will be a problem with some of our clients.

Reply:
maybe we should sanitize the html to be safe? `https://www.npmjs.com/package/dompurify`